Lib/test/test_importlib/extension/test_loader.py

```diff
@@ -29,6 +29,24 @@ def test_load_module_API(self):
         with self.assertRaises(ImportError):
             self.load_module('XXX')
 
+    def test_deprecation_warning_argument_name_and_fullname_use(self):
+        with self.assertWarns(DeprecationWarning) as cm:
+            self.machinery.ExtensionFileLoader(
+                    name=util.EXTENSIONS.name,
+                    path=util.EXTENSIONS.file_path)
+
+        self.assertIn("fullname", str(cm.warning))
+
+    def test_argument_name_and_fullname_use(self):
+        with self.assertWarns(DeprecationWarning) as warning_cm, \
+              self.assertRaises(TypeError) as exception_cm:
+            self.machinery.ExtensionFileLoader(
+                fullname=util.EXTENSIONS.name,
+                name=util.EXTENSIONS.name,
+                path=util.EXTENSIONS.file_path)
```

Comment:
```suggestion
        with self.assertWarns(DeprecationWarning) as warning_cm:
            with self.assertRaises(TypeError) as exception_cm:
                self.machinery.ExtensionFileLoader(
                    fullname=util.EXTENSIONS.name,
                    name=util.EXTENSIONS.name,
                    path=util.EXTENSIONS.file_path)
```

Reply:
I was wondering which one would be the better approach. I went with the former one. Thanks for this one.